scripts/build.sh

```diff
@@ -17,7 +17,7 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
-GITVERSION=`git describe --always`
+GITVERSION=`git describe --always --exact-match 2> /dev/null || echo "$(git symbolic-ref HEAD 2> /dev/null | cut -b 12-)-$(git log --pretty=format:"%h" -1)"`
```

Comment:
This [commit](https://github.com/intelsdi-x/snap/blob/master/scripts/build.sh#L20) already made it into master. I think you can safely rebase it out of this PR.